cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -576,17 +620,18 @@ and from within a workflow with a predicate, fork and joins::
     
     String startTime = token.get("start.time", "PurchaseHistoryBuilder");
  
-    // To get the most recent value of counter with group name
+    // To get the most recent value of the counter with group name
     // 'org.apache.hadoop.mapreduce.TaskCounter' and counter name 'MAP_INPUT_RECORDS':
    
-    String flattenCounterKey = "mr.counters.org.apache.hadoop.mapreduce.TaskCounter.MAP_INPUT_RECORDS";
-    workflowToken.get(flattenCounterKey, WorkflowToken.Scope.SYSTEM);
+    String flattenCounterKey = "org.apache.hadoop.mapreduce.TaskCounter.MAP_INPUT_RECORDS";
+    int records = workflowToken.get(flattenCounterKey, WorkflowToken.Scope.SYSTEM).getAsInt();
  
-    // To get the value of counter with group name
+    // To get the value of the counter with group name
     // 'org.apache.hadoop.mapreduce.TaskCounter' and counter name 'MAP_INPUT_RECORDS' as
-    // set by a MapReduce program with unique name 'PurchaseHistoryBuilder':
+    // set by a MapReduce program with the unique name 'PurchaseHistoryBuilder':
```

Comment:
is the "unique name" the unique node id within the workflow? I don't think it is clear here what "unique" means.